src/osdproc.c

```diff
@@ -692,79 +708,155 @@ void draw_time() {
                SIZE_TO_FONT[eeprom_buffer.params.Time_fontsize]);
 }
 
-void draw_CWH(void) {
-  char tmp_str[100] = { 0 };
-  float dstlon, dstlat, dstsqrt;
+// Haversine distance
+// http://www.movable-type.co.uk/scripts/latlong.html
+float get_distance_between_locations_in_meters(float lat_one, 
+                                               float lon_one, 
+                                               float lat_two, 
+                                               float lon_two)
+{
```

Comment:
currently we have the opening `{` on the line of the function definition (which I prefer) but we could talk about general code standards like this if you care.

I started using uncrustify but it does some odd things every now and then.

I used clang_format quite a bit lately and it seems to work nicely.
It formats stuff a bit weird (for my taste at least), but it is consistent and fast.

My suggestion would be to ignore style things like this, decide on a tool to enforce styling, choose a style and just reformat all our code at once